<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VerifyMemberForm</class>
 <widget class="QWidget" name="VerifyMemberForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>993</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="TitleLabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>210</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>Verification</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="ConfigurationFrame">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>290</y>
     <width>751</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QRadioButton" name="SystemUserRadioButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>System User</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="EmployeeRadioButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Employee</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SAIDLineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="placeholderText">
     <string>South African 13 Digit ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SearchPushButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Search Member</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/resources/icons/search.png);
background-repeat: no repeat;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="MemberRadioButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>97</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Member</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="LogoAppLabel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>50</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resources/icons/taxi.png);
background-repeat: no repeat;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
